server/handlers_test.go

```diff
@@ -69,6 +70,7 @@ func TestHTTPServer_ReportEndpoint(t *testing.T) {
 }
 
 func TestHTTPServer_ReportEndpoint_WithOnlyOSDEndpoint(t *testing.T) {
+	time.Sleep(1 * time.Second)
```

Comment:
that's strange

Reply:
it is, but it lowers the chance to get unit test failure. We have to use different approach though (this does not fix DL, of course)